{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "0214c459_70813c72",
        "filename": "src/net/http/serve_test.go",
        "patchSetId": 13
      },
      "lineNbr": 793,
      "author": {
        "id": 6365
      },
      "writtenOn": "2022-11-22T18:36:29Z",
      "side": 0,
      "message": "I think this drops the `close` call on the `case \u003c-errc` path..?",
      "revId": "85a2c19b328081c3fbcd1fa3db9a56d708a25c68",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705"
    }
  ]
}